coss/opensource_clubs/models.py

```diff
@@ -45,17 +45,33 @@ def __str__(self):
 class HomePage(Page):
     """Landing HomePage for OpenSource clubs."""
 
+    logo = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
     heading_text = fields.RichTextField(verbose_name='Text', blank=True, default='',)
     heading_cta_text = models.CharField(verbose_name='CTA Text', blank=True, default='',
                                         max_length=50)
     heading_cta_link = models.URLField(verbose_name='CTA Link', blank=True, default='')
+    heading_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
     discourse = models.ForeignKey(
         'discourse.DiscourseCategory',
         null=True,
         blank=True,
         on_delete=models.SET_NULL,
         related_name='+'
     )
+    show_more_text = models.CharField(verbose_name='"Show more" text', default='Show more',
+                                      max_length=250)
```

Comment:
The `Show more` link is generated from the entity page. See here [0]. Due to that you can drop this completely. 

[0] https://github.com/mozilla/coss/blob/master/coss/opensource_clubs/templates/opensource_clubs/featured_club_card.html